lldp/tx.c

```diff
@@ -96,9 +96,10 @@ bool mibConstrInfoLLDPDU(struct port *port, struct lldp_agent *agent)
 	fb_offset += ptlv->size;
 	free_pkd_tlv(ptlv);
 
-	if (datasize < ETH_MIN_DATA_LEN)
+	if (datasize < ETH_MIN_DATA_LEN) {
+		memset(agent->tx.frameout + fb_offset, 0, ETH_ZLEN - fb_offset);
```

Comment:
This fix looks good.  I guess we have the same issue in mibConstrShutdownLLDPDU as well.  Do you agree?

Reply:
Yes, I'll fix that too.